### ISSUE TITLE
The ADD_test metric is low during training process

### ISSUE THREAD

**Issue by YuukkiFu** [lyltc1/HiPose#4]
After training the code you provided, the output images for the test and train sets, as shown in the figure, indicate that the test accuracy is only around 0.03, which is significantly different from the results in the paper.
![截图 2024-07-05 16-20-22](https://github.com/lyltc1/HiPose/assets/147615864/4db08f42-2e25-45a1-a213-8ab04ca04c63)
![截图 2024-07-05 16-19-24](https://github.com/lyltc1/HiPose/assets/147615864/4ac91409-c714-40bd-aedc-faf9bb522a54)

 I also imported the weights you provided into the training code and found that the output accuracy is also very low. I am wondering if there might be some issues with the training code?
![1](https://github.com/lyltc1/HiPose/assets/147615864/86fb0283-c15d-4cea-9a5c-9fa765817dae)

**Comment by lyltc1** [comment]
Thank you for pointing that out. A possible reason for this issue is the presence of too many incorrect bounding boxes in the detection files, which can reduce the accuracy of the test data in training process. However, you can disregard this and compare the results using the BOP score, which serves as a fair benchmark.

**Comment by YuukkiFu** [comment]
Thank you for your suggestions. However, the results in your paper also involve precision testing on the original dataset. Currently, I would like to reproduce this result. Do you have any ideas for improvement? In addition, the provided code has only been verified in the BOP 300 test set. Is this result also obtained in the BOP test set? Or in all test image？
![image](https://github.com/lyltc1/HiPose/assets/147615864/896872dd-08c8-484f-a1a2-096f53a779f9)

**Comment by lyltc1** [comment]
Sorry for the late reply. Setting bop_challenge = False in the configuration file ([test_ycbv_config.txt](https://github.com/lyltc1/HiPose/blob/main/hipose/config/test_ycbv_config.txt)) will switch to using all test image, as opposed to the default behavior when bop_challenge = True.

**Comment by YuukkiFu** [comment]
Thanks!!! I have got it.
